driver/src/util.rs

```diff
@@ -8,6 +8,24 @@ use web3::types::{H256, U256};
 
 const BATCH_TIME_SECONDS: u32 = 3 * 60;
 
+pub trait CeiledDiv {
+    fn ceiled_div(&self, divisor: Self) -> Self;
+}
+
+impl CeiledDiv for u128 {
+    fn ceiled_div(&self, divisor: u128) -> u128 {
+        //ceil(p / float(q)) == (p + q - 1) / q
+        (self + divisor - 1) / divisor
+    }
+}
+
+impl CeiledDiv for U256 {
+    fn ceiled_div(&self, divisor: U256) -> U256 {
```

Comment:
Gonna test these?

Reply:
<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia3.giphy.com%2Fmedia%2F3o6ZtqPYkSprj7nJUA%2Fgiphy.gif&token=caught_me" data-gitmeme-token="caught_me"><img src="https://media3.giphy.com/media/3o6ZtqPYkSprj7nJUA/giphy.gif" title="Created by gitme.me with /caught_me"/></a>

Reply:
I would recommend the tests in the branch I created earlier. Should be called ceiling quotient.